resources/public/api/conf/1.0/schemas/broughtForwardLosses/listBFLosses/list_response.json

```diff
@@ -28,7 +28,7 @@
                   "example": "/individuals/losses/{nino}/losses-brought-forward/{lossId}"
                 },
                 "rel": {
-                  "description": "A label for the endpoint, which describes how it is related to the current resource\nThe rel will be `self` where the action is retrieval of the same resource",
+                  "description": "A label for the endpoint, which describes how it is related to the current resource. The rel will be self where the action is retrieval of the same resource",
```

Comment:
Should we say here that the links are related to the list item (not the full list resource)...?

Reply:
Myself and Luke are happy with the generic-ness of this wording